example/src/AccountOverviewScreen.tsx

```diff
@@ -111,7 +111,7 @@ export const AccountOverviewScreen = (props: { rlyAccount: string }) => {
             <View style={styles.alignMiddle}>
               <BodyText>Register My Account</BodyText>
             </View>
-            <Button onPress={registerAccount} title="Register" />
+            <Button onPress={claimRlyTokens} title="Register" />
```

Comment:
Should we make this "Claim Tokens" also or does that make our sign up page look like a spam website lol

Reply:
That's a good point. Makes sense from a consistency standpoint